admin_manual/installation/source_installation.rst

```diff
@@ -2,47 +2,177 @@
+     for VM specific installation and usage details).
+* NextcloudPi (aka: NCP)
+   - A community-driven installation method that is actively maintained by amazing Nextcloud community members 
+     (make sure to thank them - or find a way help them out - if you get the opportunity!).
+   - A ready to use image for Virtual Machines, Raspberry Pi, Odroid HC1, Rock64 and other boards. Also supports
+     LXD and LXC containers and there is an install script for the latest supported Debian based system as well.
+   - It will setup everything for you and include scripts for automated installation of apps like: Collabora, 
+     OnlyOffice, Talk and so on. Full feature list available 
+     `here <https://github.com/nextcloud/nextcloudpi?tab=readme-ov-file#features>`_.
+   - `Official Documentation <https://github.com/nextcloud/nextcloudpi>`_ (**Note**: see Official Documentation 
+     for NCP specific installation and usage details).
+* Nextcloud Community Docker Image
+   - A community-driven installation method that is actively maintained by amazing Nextcloud community members 
+     (make sure to thank them - or find a way help them out - if you get the opportunity!).
+   - Two editions of the image are provided for building different types of Nextcloud stacks: the Apache image 
+     contains a full Nextcloud installation including an Apache web server. The second option is an FPM 
+     installation and runs a FastCGI process that serves your Nextcloud installation. The standard images are
+     Debian-based, but Alpine variants are offered as well.
+   - This image is designed for expert use and intended to be used in a micro-service environment where you can
+     incorporate as a building block in your own Nextcloud stack (i.e. you will not need to install/update 
+     Nextcloud, but you will need to supply your preferred web, reverse proxy, HTTPS termination, database and 
+     other desired supplementary services).
+   - Full Nextcloud stacks, incorporating the image, are provided in the form of example Docker Compose files 
+     and Dockerfile variations. Full feature list available `here <https://github.com/nextcloud/docker/>`_.
+   - `Official Documentation <https://github.com/nextcloud/docker/>`_ (**Note**: see Official Documentation 
+     for Image specific installation and usage details).
+* Nextcloud Web Installer
+   - The Web Installer is an easy way to install Nextcloud Server in a shared / managed web space (e.g. shared 
+     hosting) if you don't have access to the command line and if your environment meets the requirements.
+   - Checks for essential PHP dependencies, downloads the Nextcloud Archive file from the official server, and 
+     unpacks it with the right permissions then directs you to the Nextcloud Server Setup Wizard.
+   - `Official Documentation <https://github.com/nextcloud/web-installer/>`_
+* One Click Signup
```

Comment:
You don't really install anything. What must be clear, in contrast to the self-installed versions, you are not in control of these setups. You need to trust these providers as you would have to trust any other cloud-storage provider.
There were examples of providers shutting down service on short notice leaving behind many frustrated users with life-time contracts. And if their service fails, they end up at the Nextcloud community, and ask for help. And there is little guidance to check for "good" service providers. Nextcloud says to select quality providers, but how is this done? I think it would be better to say, what is done that people don't expect too much.

Perhaps it is also a nice way, to have a "look and feel" of Nextcloud and try a bit.